scripts/valhalla_build_extract.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+import argparse
+from io import BytesIO
+import json
+import logging
+from pathlib import Path
+import struct
+import sys
+import tarfile
+from tarfile import BLOCKSIZE
+from typing import Dict
+
+description = """Builds a tar extract from the tiles in mjolnir.tile_dir to the path specified in mjolnir.tile_extract."""
+
+# "=" prefix means native byte order with standard size and no alignment:
+# https://docs.python.org/3/library/struct.html#byte-order-size-and-alignment
+STRUCT_FORMAT = '=LQ'
```

Comment:
this confuses me a little, lacking the solid CS background I guess.. this is the byte layout specifier for `struct` for `uint32_t` and `uint64_t` which creates 12 byte packets for two numbers: https://docs.python.org/3/library/struct.html#byte-order-size-and-alignment, the comment says it all. is this the way how to make sure any consumer architecture can read any other producer's architecture's `index.bin` file with this layout (e.g. I produce an extract on `x86_64` and someone with `ppc64` can read it)? or is there some more trickery involved in python or c++ to make sure endianess etc is properly taken care of?